packages/ag-grid-theme/webpack.config.js

```diff
@@ -0,0 +1,43 @@
+const path = require("path");
+const MiniCssExtractPlugin = require("mini-css-extract-plugin");
+const CopyPlugin = require("copy-webpack-plugin");
+
+module.exports = {
+  mode: "production",
+  entry: "./src/index.js",
+  output: {
+    filename: "index.js",
+    path: path.resolve(__dirname, "dist"),
+  },
+  plugins: [
+    new CopyPlugin({
+      patterns: [
+        { from: "src/fonts", to: "assets/fonts" },
+        { from: "src/icons", to: "assets/icons" },
+        { from: "src/images", to: "assets/images" },
+      ],
+    }),
+    new MiniCssExtractPlugin({
```

Comment:
@NicDominguez I know we talked about making this work in webpack, but have you looked at [Rollup?](https://rollupjs.org/guide/en/) It's what Stencil uses under the hood, and would likely be a lot easier.